tests/conftest.py

```diff
@@ -233,6 +234,14 @@ class TestEnv:
     # print(f"deleted virtual environment in {venv_dir}\n")
 
 
+@pytest.fixture(scope="session")
+def graphics_test_mechdb_file():
+    """Download mechdb files for graphics export test."""
+    mechdb_file = download_file("graphics_test.mechdb", "pymechanical", "test_files")
+    yield mechdb_file
+    os.remove(mechdb_file)
```

Comment:
@dipinknair why remove it here? Can't it be reused so it doesn't download again each time the test is run?

Reply:
@koubaa , Oh I thought you suggested only to reuse for Mutiple tests in same session. I will remove this line of code so that it can be reusable for multiple sessions when we run pytest.